packages/patternfly-4/react-core/src/components/Tabs/Tabs.js

```diff
@@ -264,72 +174,85 @@ class InternalTabs extends React.Component {
-          <button
-            className={css(styles.tabsScrollButton)}
-            variant="plain"
-            aria-label={leftScrollAriaLabel}
-            onClick={this.scrollLeft}
-          >
-            <AngleLeftIcon />
-          </button>
-        )}
-        <ul className={css(styles.tabsList)} ref={this.tabList} onScroll={this.handleScrollButtons}>
-          {children.map((child, index) => (
-            <li
-              key={index}
-              className={css(
-                styles.tabsItem,
-                child.props.eventKey === activeKey && styles.modifiers.current,
-                className
-              )}
+        <Component
+          aria-label={ariaLabel}
+          className={css(
+            styles.tabs,
+            isFilled && styles.modifiers.fill,
+            isSecondary && styles.modifiers.tabsSecondary,
+            showLeftScrollButton && styles.modifiers.start,
+            showRightScrollButton && styles.modifiers.end,
+            highlightLeftScrollButton && styles.modifiers.startCurrent,
+            highlightRightScrollButton && styles.modifiers.endCurrent,
+            className
+          )}
+          {...props}
+        >
+          {!isSecondary && (
+            <button
+              className={css(styles.tabsScrollButton)}
+              variant="plain"
+              aria-label={leftScrollAriaLabel}
+              onClick={this.scrollLeft}
             >
-              <Tab
-                {...child.props}
-                ref={node => {
-                  this.child = node;
-                }}
-                className={css(styles.tabsButton)}
-                onClick={event => this.handleTabClick(event, child.props.eventKey, child.props.tabContentRef)}
-                id={`pf-tab-${child.props.eventKey}-${child.props.id || id}`}
-                aria-controls={
-                  child.props.tabContentId
-                    ? child.props.tabContentId
-                    : `pf-tab-section-${child.props.eventKey}-${child.props.id || id}`
-                }
+              <AngleLeftIcon />
+            </button>
+          )}
+          <ul className={css(styles.tabsList)} ref={this.tabList} onScroll={this.handleScrollButtons}>
+            {children.map((child, index) => (
+              <li
+                key={index}
+                className={css(
+                  styles.tabsItem,
+                  child.props.eventKey === activeKey && styles.modifiers.current,
+                  className
+                )}
               >
-                {child.props.title}
-              </Tab>
-            </li>
-          ))}
-        </ul>
-        {!isSecondary && (
-          <button
-            className={css(styles.tabsScrollButton)}
-            variant="plain"
-            aria-label={rightScrollAriaLabel}
-            onClick={this.scrollRight}
-          >
-            <AngleRightIcon />
-          </button>
+                <Tab
+                  {...child.props}
```

Comment:
Can we stay consistent and propogate at the end?

Reply:
yes we should thanks